### DISCUSSION TITLE
Quantify variance explained by individual factors on the Principle components.

### DISCUSSION THREAD

**Discussion by annrosebright** [GreenleafLab/ArchR#1408] [Questions / Documentation]
Hi,

I have different stage-specific scATACseq datasets. An analysis I would like to perform is to quantify the variance by factors ( distal peak, promoters. gene bodies) on the various PC components.  How much variance does the first PC explain? Is there a way to extract this information? 
Sorry if I have not explained it well. 
Thanks in advance.

**Comment by rcorces** [comment]
In scATAC, we dont have PCs per se. LSI and SVD are slightly different, but you can obtain variance explained by the various singular values of SVD. I do not have code to do this but google may help.
